### DISCUSSION TITLE
How to verify enrollement for remote program?

### DISCUSSION THREAD

**Discussion by alexdhaynes** [community/community#116686] [GitHub Education]
### Hi everyone,

I'm enrolled in a program remotely. My documentation was not accepted for the following reason: "You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote."

I don't have any documentation that specifies virtual learning. Is there another way I can upload supporting documentation to confirm my current enrollment?

**Comment by samus-aran** [comment]
Hi @alexdhaynes 👋 

Is that your only rejection reason? I would suggest you take another live capture of your evidence and ensure text is legible. Let us know how you get on again 👍

**Reply by alexdhaynes** [reply]
Ok will do, thank you!

**Comment by ibrxhim737** [comment]
I'm facing a similar problem, all I have are emails. My Uni ID did not work and thats all ive got unfortunately.
